server/channels/db/migrations/postgres/000111_create_retentionidsfordeletion_table.up.sql

```diff
@@ -0,0 +1,7 @@
+CREATE TABLE IF NOT EXISTS retentionidsfordeletion (
+    id varchar(26) PRIMARY KEY,
+    tablename varchar(64),
+    ids varchar(26)[]
+);
+
+CREATE INDEX IF NOT EXISTS idx_retentionidsfordeletion_tablename ON retentionidsfordeletion (tablename);
```

Comment:
@agnivade This throws an error: `CREATE INDEX CONCURRENTLY cannot run inside a transaction block`. I don't think it matters that we are locking this new table when creating the index because it's going to be empty anyway? Can I remove the `CONCURRENTLY`?

Reply:
Woops, my bad, I wrote the feature and I forgot it. You have to add a comment `-- morph:nontransactional` on the line before `CREATE INDEX CONCURRENTLY`. Then it should work.

Reply:
Sorry, to clarify, when you add the `-- morph:nontransactional` comment, the SQL queries inside the file run without a transaction. So what we need to do here is that split the migration into 2 files: one for create table, another for creating the index. And then add the comment on top of the 2nd file. But I do agree with your point that there's not much point to this elaborate dance for a new table. So fine by me if you want to skip it.